sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -433,6 +456,73 @@ Construct a SYCL buffer instance from a pointer to a Level Zero memory allocatio
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
 </tr>
+
+<tr>
+<td>
+
+``` C++
+template<backend Backend, int Dimensions = 1, typename AllocatorT = sycl::image_allocator>
+image<Dimensions, AllocatorT> make_image(
+    const backend_input_t<Backend, image<Dimensions, AllocatorT>> &backendObject,
+    const context &targetContext);
+```

```

Comment:
Thanks for adding the template definition.  This makes the prototype of the function much clearer.  Two comments, though:

1. The HTML render on my monitor (1920 x 1200) has a horizontal scroll bar, which makes the table hard to read.  (You need to scroll left and right to read the "description" column.)  This is caused because the synopses for `make_image` use more columns than the other synopses in this table.  Can you add some line breaks to these synopses to avoid this?  For example, maybe:

   ```
   template<backend Backend, int Dimensions = 1,
            typename AllocatorT = sycl::image_allocator>
   image<Dimensions, AllocatorT> make_image(
       const backend_input_t<Backend,
                             image<Dimensions, AllocatorT>> &backendObject,
       const context &targetContext);
   ```

2. Please update the other entries in the table to have the `template` keyword, template parameters, and return type to match the style of the `make_image` entries.

Reply:
FWIW, my HTML preview in VS code does not insert scroll bars. I'll add more line breaks though.